Wikipedia/Code/Session.swift

```diff
@@ -261,24 +267,24 @@ import Foundation
         return task
     }
 
-    public func jsonDecodableTask<T: Decodable>(with url: URL?, method: Session.Request.Method = .get, bodyParameters: Any? = nil, bodyEncoding: Session.Request.Encoding = .json, authorized: Bool? = nil, completionHandler: @escaping (_ result: T?, _ response: URLResponse?,_ authorized: Bool?,  _ error: Error?) -> Swift.Void) {
```

Comment:
Apple's example in the Type Constraint Syntax part of The Swift Programming Language (4.2) https://docs.swift.org/swift-book/LanguageGuide/Generics.html shows the `<T: Decodable>` syntax as an example, I think it's more readable to have it up front, especially if the signature is long

Reply:
good point - fixed